users/urls_api.py

```diff
@@ -2,8 +2,8 @@
 from . import views_api
 
 urlpatterns = [
-    path('users/', views_api.UsersView.as_view()),
-    path('users/me/', views_api.UserDetailView.as_view()),
-    path('auth/login/', views_api.LoginView.as_view()),
-    path('auth/change-password/', views_api.ChangePasswordView.as_view()),
+    path('users/', views_api.UsersView.as_view(), name="api-create-user"),
+    path('users/me/', views_api.UserDetailView.as_view(), name="api-edit-user"),
+    path('auth/login/', views_api.LoginView.as_view(), name="api-login"),
+    path('auth/change-password/', views_api.ChangePasswordView.as_view(), name="api-change-password"),
```

Comment:
Let's add namespace to urls imported from users module. That way you will no longer need to add api prefix to url names.

You can use `api-users` for a namespace.

https://docs.djangoproject.com/en/2.2/topics/http/urls/#url-namespaces